src/wasm/kotlin/javatests/arcs/EntityClassApiTest.kt

```diff
@@ -63,13 +63,13 @@ class EntityClassApiTest(ctor: (String) -> EntityClassApiTest_Errors): TestBase<
     @Test
     fun testEncodingDecoding() {
         val empty = EntityClassApiTest_Data()
-        val emptyStr = empty.encodeEntity()
-        val decodedEmpty = EntityClassApiTest_Data().decodeEntity(emptyStr)
+        val encodedEmpty = empty.encodeEntity()
+        val decodedEmpty = EntityClassApiTest_Data().decodeEntity(encodedEmpty.bytes)
         assertEquals("Encoding and Decoding an empty entity results in the same entity", empty, decodedEmpty)
 
         val full = EntityClassApiTest_Data(num=10.0, txt="20", lnk="https://thirty.net", flg=true)
```

Comment:
Spaces are preferred between param names and values.

Reply:
Fixed.